### PR TITLE
Fix OpenAPI Table Writeset Spec + Generate new TS SDK version with latest API changes

### PR DESCRIPTION
**Pull request by CapCap**
The OpenAPI yaml didn't match the node, and we should generate a new TS SDK to ensure the latest and greatest are out there for our community.

Due to the renaming of `UserCreateSigningMessageRequest` in the openAPI, this is a breaking change, and so I've bumped up the major version a per SEMVER.

### CODE REVIEW

File: ecosystem/typescript/sdk/package.json [hunk 1]
```diff
@@ -54,5 +54,5 @@
     "tweetnacl": "^1.0.3",
     "yarn": "^1.22.18"
   },
-  "version": "0.0.20"
+  "version": "1.0.0"
```
Comment (davidiw) [inline ecosystem/typescript/sdk/package.json:57]: wow, not even 0.1.0? 1.0 seems to be better reserved for our mainnet?

**Review by davidiw — APPROVED** [review]

File: ecosystem/typescript/sdk/package.json (hunk 1 above)
Comment (CapCap) [reply ecosystem/typescript/sdk/package.json:57]: In semver it seems that only the top level number is for breaking changes

**Pull request closed without merging**